### DISCUSSION TITLE
API Google Search

### DISCUSSION THREAD

**Discussion by TheAlexSandro** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#329] [General]
Ada yang punya API google search?

**Comment by Mas88-hub** [comment]
> https://api.ngodingaja.my.id/api/gsearch?search=milfu%20bot
> 
> Gini?

**Comment by LemonSync** [comment]
> Ada yang punya API google search?

Gw rasa, google gak nyediain api untuk search engine. Tapi lu bisa cek https://developers.google.com/custom-search/v1/overview?hl=id 
Karna mungkin tertera penyediaan api disitu
